lib/shark_on_lambda/rspec/helpers.rb

```diff
@@ -93,6 +103,12 @@ def make_request(method, action, options = {})
         errors = env['rack.errors']
         @response = Rack::MockResponse.new(status, headers, body, errors)
       end
+
+      def path_info?(action)
+        action.is_a?(String) && URI.parse(action).path.present?
+      rescue
+        false
+      end
```

Comment:
Question about all these changes in `rspec/helpers`.

Do we want this `path_info` stuff in these helpers or do we want to create new ones? 
What should be the DSL then, because `get '/foo/bar/123, headers: { ... }'` feels kinda nice.